redundant-bmc/src/sibling_impl.cpp

```diff
@@ -332,4 +332,43 @@ sdbusplus::async::task<> SiblingImpl::waitForSiblingRole()
     co_return;
 }
 
+// NOLINTBEGIN(clang-analyzer-core.uninitialized.Branch,-warnings-as-errors,
+//             readability-static-accessed-through-instance,-warnings-as-errors)
+sdbusplus::async::task<> SiblingImpl::waitForBMCSteadyState() const
+{
+    using namespace std::chrono_literals;
+    auto start = std::chrono::steady_clock::now();
+    std::chrono::minutes timeout{10};
+    bool waiting = false;
+
+    // If sibling isn't alive don't bother waiting
+    if (!hasHeartbeat())
```

Comment:
Do we need some sort of small delay just to be sure the other BMC isn't just a second or two behind the boot of this one?

Reply:
When it's first coming up, it goes through Manager::startup(), which is where it waits for up to 6 minutes for the sibling's heartbeat to start in waitForSibingUp().  So by the time it gets here, I don't think there's a need to wait anymore.